meta-lmp-bsp/dynamic-layers/freescale-layer/recipes-bsp/imx-atf/imx-atf_%.bbappend

```diff
@@ -3,6 +3,10 @@ FILESEXTRAPATHS:prepend := "${THISDIR}/${BPN}:"
 PROVIDES += "virtual/trusted-firmware-a"
 RPROVIDES:${PN} += "virtual/trusted-firmware-a"
 
+SRC_URI:append = " \
```

Comment:
I'm double-checking if the patch is for any machine because the patch looks like to be only for imx8m.

Reply:
technically it doesn't affect any other platforms at all. But I can add an explicit tag

Reply:
Since it covers 2 machines and already patching a bsp-specific recipe (imx-atf), I think it is fine to have it applied for all.